README.md

```diff
@@ -17,76 +16,119 @@ _A table of contents to help people find things_
+A variation of the alert message is an **inline** alert message, which has almost exactly the same markup, with an optional addition of information, and does not have the option to close the message.
 
 ```html
-<div data-o-component="o-component-boilerplate" class='o-component-boilerplate'>
+<div class="o-message o-message--alert--inline o-message--alert-success" data-o-component="o-message">
+	<div class="o-message__container">
+		<div class="o-message__content">
+			<p class="o-message__content--highlight">Hooray!
+				<span class="o-message__content--detail">The quick brown fox jumped over the lazy dogs!</span>
+			</p>
+			<p class="o-message__content--additional-info">Did you know that that sentence uses all of the letters in the alphabet at least once?</p>
+
+			<div class="o-message__actions">
+				<a href="#" class="o-message__button o-message__action--primary">Button</a>
+				<a href="#" class="o-message__link o-message__action--secondary">Text link</a>
+			</div>
+		</div>
+	</div>
 </div>
 ```
 
 ### JavaScript
-_Remember to start your codeblocks with three backticks and "js" so your js is syntax highlighted correctly._
-
-_Though it's not practical to repeat every aspect of Origami modules convention for every component, **A LOT** of people get tripped up by modules not auto initialising, so this line (remember to change the `o-component-boilerplate` to your component name) is useful if you have JavaScript:_
+No code will run automatically unless you are using the Build Service. You must either construct an `o-message` object or fire an o.DOMContentLoaded event, which `o-message` listens for.
 
-No code will run automatically unless you are using the Build Service.
-You must either construct an `o-component-boilerplate` object or fire the `o.DOMContentLoaded` event, which oComponent listens for.
 
-#### Constructing an o-component-boilerplate
+#### Constructing an o-message
 
+If you have set up your message declaratively, and are using default o-message classes, use the following:
 ```js
-const oComponentBoilerplate = require('o-component-boilerplate');
-
-const oComponentBoilerplate = new oComponentBoilerplate();
+const oMessage = require('o-message');
+const messageElement = document.getElementById('my-message');
+const importantMessage = new oMessage(messageElement, { content: { highlight: 'Success' } });
 ```
+The second argument that `oMessage` accepts is an (options object)[#options], which can be used to change some behaviour and some markup of a message.
 
-#### Firing an oDomContentLoaded event
+If you're setting up a message without existing DOM elements, oMessage will construct an element for you when it is set up like this:
 
 ```js
-document.addEventListener('DOMContentLoaded', function() {
-	document.dispatchEvent(new CustomEvent('o.DOMContentLoaded'));
+const oMessage = require('o-message');
+const importantMessage = new oMessage(null, {
+	theme: 'error',
```

Comment:
Thoughts on switching "theme" to a more brandy "variant"?

Reply:
Wouldn't a variant be more accurate in describing that an alert is `inline` or `bleed`?